gnomad/utils/vcf.py

```diff
@@ -1158,6 +1233,7 @@ def make_vcf_filter_dict(
 def make_hist_bin_edges_expr(
     ht: hl.Table,
     hists: List[str] = HISTS,
+    ann_with_hists: str = "",
```

Comment:
I know we pass other empty strings but I think given how this one works in the conditional, it makes more sense to make it 
```suggestion
    ann_with_hists: Optional[str] = None,
```
This works fine calling the function on its own for me.

Reply:
I can't suggest it here because you havent changed them at all but the Optional will need to be added to the typing import at the beginning of the module.